cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java

```diff
@@ -194,6 +206,10 @@ public void execute() throws CommandException {
 
     repository = commandHelper.findRepositoryByName(repositoryParam);
 
+    if (isParallel) {
```

Comment:
Just wondering... at this point, is there really a value in keeping the 2 implementation mixed, vs having to distinct path as it is seem there are quite a few "isParallel"? I can image that we'll be removing the non-parallel eventually?

Reply:
I don't believe the should be separate cli commands as there's quite a bit of crossover between pull run recordings, locale mapping etc but I can look into moving the parallel specific implementation into a separate class which is called from the pull command execute method, then we'd just require a single if statement in the PullCommand to call this new class

Reply:
Pushed an update that moves the parallel logic to a separate class that is called from the PullCommand if `--parallel` flag is passed